frontend/model/state.js

```diff
@@ -67,6 +67,15 @@ sbp('okTurtles.events/on', EVENTS.CONTRACT_IS_SYNCING, (contractID, isSyncing) =
   contractIsSyncing[contractID] = isSyncing
 })
 
+const isInMochaTest = typeof globalThis.it === 'function'
+
+if (isInMochaTest) {
+  sbp('sbp/selectors/unsafe', [
+    'state/enqueueHandleEvent',
+    'state/vuex/state'
+  ])
+}
+
```

Comment:
Thanks @taoeffect, @snowteamer 

I thought these lines might need your look too. 
Have added this change for L:63 in `~/test/backend.test.js` which is another place where `sbp/selectors/overwrite` is called.
(used `globalThis` instead of `global` in node so that this script doesn't fail in browser. )

what's your thoughts?